charts/taiga/values.yaml

```diff
@@ -7,11 +7,11 @@ replicaCount: 1
 # TFLogin customized images
 images:
   backend:
-    repository: threefolddev/tf-taiga-back
+    repository: threefolddev/taiga-back-threefold
```

Comment:
do we still need these ones? aren't they outdated and we agreed to be aligned with the upstream? or I'm reading it wrong?

Reply:
These depend on the latest original taiga image and just install the plugin on top of it.